lib/processors/bundlers/moduleBundler.js

```diff
@@ -152,14 +152,12 @@ export default function({resources, options: {
 		ignoreMissingModules: false
 	}, bundleOptions);
 
+	// Take the project from the resources if available.
+	// It's needed for specVersion check. If unavailable,
+	// the specVersion would be treated as the most recent one.
+	const project = resources?.[0]?.getProject();
```

Comment:
I'm afraid that `resources` might contain resources from multiple project, not only the one currently being built. Think for example of an app generating a self-contained bundle.

Similar to other cases, we should do the specVersion check in the individual tasks (in this case all the bundling tasks) and set the configuration there.

Reply:
Ah, I see!

Thanks a lot! I'll try to cover this

Reply:
Now the `taskUtil` is part of `moduleBundler`'s signature as we do that in other places